### ISSUE TITLE
bug: Page jump top after child page (overflow-x: scroll)

### ISSUE THREAD

**Issue by roman-rr** [ionic-team/ionic#19644]
# Bug Report

**Ionic version:**
[x] **4.x**
[x] **iOS only**

**Current behavior:**
After back from child page and scroll any horizontal stack (div with overflow-x: scroll), ion-content jump to top.  
The similar scrollTop() event i found after swipe child page quickly - from left to right. 
Possible to remove this scroll jump after back to parent page ? 

**Expected behavior:**
scrollTop value of ion-content persist the value and don't reset to 0.

**Example code:**
https://github.com/roman-rr/horizontal-stacks

**Other information:**

![preview](https://user-images.githubusercontent.com/10646478/66708941-cd289b80-ed83-11e9-8941-d924fabe4f36.gif)

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.2.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.9.0
   @angular-devkit/build-angular : 0.803.5
   @angular-devkit/schematics    : 8.3.5
   @angular/cli                  : 8.3.5
   @ionic/angular-toolkit        : 2.0.0

Cordova:

   Cordova CLI       : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms : ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.1.2, (and 17 other plugins)

Utility:

   cordova-res : 0.4.0 (update available: 0.8.0)
   native-run  : 0.2.4 (update available: 0.2.8)

System:

   Android SDK Tools : 26.1.1 (/Users/romanrr/Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 8.0.2
   NodeJS            : v10.16.0 (/usr/local/bin/node)
   npm               : 6.9.0
   OS                : macOS Catalina
   Xcode             : Xcode 11.0 Build version 11A420a
```

**Comment by roman-rr** [comment]
The similar scrollTop event i found when swipe child page quickly from left to right. 
Possible to remove this scroll jump after back to parent page ?

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you provide a repo with the code required to reproduce this issue?

**Comment by roman-rr** [comment]
@liamdebeasi 
Please review asap, this is fatally stuck application release: 
https://github.com/roman-rr/horizontal-stacks
`npm i && ionic cordova bulid ios`

**Comment by liamdebeasi** [comment]
Thanks for the follow up. I am not able to reproduce this issue using the steps you provided. Can you clarify the steps to reproduce?

**Comment by thomasblom** [comment]
Hi @liamdebeasi, I have tried it with the repo @roman-rr has given. I get the same result, a scrollToTop behaviour after navigating back and sliding the carousel.

![ionic-bug-scroll](https://user-images.githubusercontent.com/8591153/66922274-06cd0100-f027-11e9-8fef-8db2c684f095.gif)

**Comment by roman-rr** [comment]
@liamdebeasi 
1. `Run on any iOS device`
2. Scroll page to bottom
3. Click to any card in horizontal stack
4. Click to back button
5. Try to scroll cards in horizontal stack

**Comment by svangasse** [comment]
I'm getting a very similar issue which I initially narrowed down to ion-content too. But further digging shows that you can get the same behaviour simply with a div with overflow-y: scroll;

Navigating back from a child page and scrolling on the original page causes scroll elements to reset.

Example code:
https://github.com/svangasse/ionic-scroll-bug

Other information:
![screen-grab](https://user-images.githubusercontent.com/1622725/67018304-f2563a80-f0f2-11e9-9b1f-1240ef6fbbe0.gif)

**Comment by liamdebeasi** [comment]
Thanks for the follow up. I am able to reproduce this on a device running iOS 13.1.3. However, I am unable to reproduce this on iOS 12 as well as iOS 13.2, so I'm wondering if maybe this is a bug in iOS? Have you been able to reproduce this issue on any other platforms/software versions?

**Comment by roman-rr** [comment]
@liamdebeasi i unable to reproduce with Android. So, its iOS bug, probably with WKWebView. 
Should we duplicate issues on ionic-webview plugin repository ?

**Comment by liamdebeasi** [comment]
I can reproduce the issue in the Safari web browser too, so I don't think it's a problem with the webview plugin.

It looks like this issue is resolved in iOS 13.2, so I am going to close this issue as there is nothing actionable on our end. If the issue reappears in iOS 13.2+ please feel free to open a new issue, and we can re-evaluate. Thanks!
